### DISCUSSION TITLE
Slovenian translation

### DISCUSSION THREAD

**Discussion by thehijacker** [henrygd/beszel#344] [Support] [answered]
I would like to translate this application to Slovenian language. Where to start? I saw you used to use weblate? But I can not find this project there.

**Comment by henrygd** [comment]
That would be great! We're on Crowdin now and I've just added Slovenian as a target language.

Please take your time. I'm visiting family and likely won't have a new release out for another few weeks or so.

https://crowdin.com/project/beszel

**Reply by thehijacker** [reply]
> That would be great! We're on Crowdin now and I've just added Slovenian as a target language.
> 
> Please take your time. I'm visiting family and likely won't have a new release out for another few weeks or so.
> 
> https://crowdin.com/project/beszel

Have started with translation. Should be finished in next few days. Been playing with this and looks very good. I have used Hetrixtools in the past but this project for sure could replace it. Few questions:

1. HDD monitoring. I have exported the ENV with sda, sda1 inside and restarted process, but nothing is showned. Not even shure where it should show. Can it show SMART errors or just size?
2. I am using Homepage as dashboard and have Beszel added as shortcut. But there is no icon for it yet. Can you add it? Project is here: https://github.com/walkxcode/dashboard-icons
3. For docker I can see CPU usage but it is crowded. Can this be seperated view per docker process? Like to have CPU and MEM per each docker service that you can sort:

![image](https://github.com/user-attachments/assets/197a5fc1-3ba2-4967-a5b7-e3d16219e350)

Thanks!

**Reply by henrygd** [reply, marked as answer]
I just opened a PR for the icons.

Currently you can only display the container stats as shown in your screenshot, however you can use the filter field to highlight specific containers.

SMART is not supported yet but it's something I'd like to add in the future.

Are you using the binary? It could be an issue with configuration or the env values not being picked up.

If you're running the binary directly, try adding the env vars directly before the command:

```bash
KEY="..." EXTRA_FILESYSTEMS=sda1 ./beszel-agent
```

If you're running as a service, make sure to modify `/etc/systemd/system/beszel-agent.service`:

```ini
[Service]
Environment="EXTRA_FILESYSTEMS=sda1"
```

```bash
sudo systemctl daemon-reload
sudo systemctl enable beszel-agent.service
sudo systemctl start beszel-agent.service
```

You can also add `LOG_LEVEL=debug` to the agent. There will be a couple lines that are logged once, immediately after startup, and begin with `DEBUG Disk partitions` and `DEBUG Disk I/O diskstats`. If you paste those lines here I'll tell you exactly how to configure it.

**Comment by thehijacker** [comment]
Translated 100 %.

**Reply by henrygd** [reply]
Thank you!
